aws-iam-role-poweruser/lambda.tf

```diff
@@ -0,0 +1,57 @@
+
+locals {
+  enable_lambda = "${var.enable_lambda ? 1 : 0}"
+
+}
+// Readonly everywhere
+resource "aws_iam_role_policy_attachment" "lambda-readonly" {
+  role       = "${aws_iam_role.poweruser.name}"
+  policy_arn = "arn:aws:iam::aws:policy/AWSLambdaReadOnlyAccess"
+}
+
+
+// Write confined to regions
```

Comment:
Rather than duplicating the AWS-managed policy, why don't we combine two policies

1. the aws-managed one
2. one of ours that does a deny on StringNotEquals region

?